src/components/AppSidebar/AppSidebar.vue

```diff
@@ -325,6 +336,22 @@ export default {
 			if (this.tabs.length > 0) {
 				this.updateActive()
 			}
+		},
+
+		/**
+		 * Emit title change event to parent component
+		 */
+		onTitleInput(event) {
+			/**
+			 * Emitted on title events of the text field
+			 * @type {Event|Date}
+			 */
+			this.$emit('inputTitle', event)
+			/**
+			 * Emitted when the title value changes
+			 * @type {string|Date}
+			 */
+			this.$emit('updateTitle:value', event.target.value)
```

Comment:
if this is the only input editable here, let's just update:title, and emit an input & change event?

Reply:
Roger that!